packages/cardano-services/src/cli.ts

```diff
@@ -150,6 +151,14 @@ commonOptions(
         stringToBoolean(enableMetrics, Programs.HttpServer, ProgramOptionDescriptions.EnableMetrics)
       )
   )
+  .addOption(
+    new Option('--disable-stake-pool-metric-apy <true/false>', ProgramOptionDescriptions.DisableStakePoolMetricApy)
```

Comment:
Can we also add a description regarding why one would want to disable it?
As a user, I would enable it since it has no documented downside.

Reply:
Good suggestion! https://github.com/input-output-hk/cardano-js-sdk/pull/609/commits/f041c9c96158a643450b5f2fb0668cfdc004e1ca